buildmaster/E155_NC_9GEV_EP_G1F1RATIO/filter.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import yaml
+import numpy as np
+import glob
+
+
+def read_data(fnames):
+    df = pd.DataFrame()
+    for fname in fnames:
+        with open(fname, "r") as file:
+            data = yaml.safe_load(file)
+
+        x = float(data["dependent_variables"][1]["qualifiers"][4]["value"])
```

Comment:
Being a bit picky here! I know it is exactly the same but for absolute consistency. The neutron one should be with `1` instead.
```suggestion
        x = float(data["dependent_variables"][0]["qualifiers"][4]["value"])
```